### ISSUE TITLE
[SF][A11y][Avatar]: [Avatar icon text does not meet minimum color contrast requirement ]

### ISSUE THREAD

**Issue by tandan0813** [UI5/webcomponents#13026]
### Describe the bug

The avatar icon text does not meet the minimum required color contrast ratio of 4.5:1 and has a ratio of 4.2:1

<img width="200" height="109" alt="Image" src="https://github.com/user-attachments/assets/cb14e38a-674c-40ee-ab3d-b55e21aa9044" />

### Isolated Example

https://ui5.github.io/webcomponents-react/v2/?path=/docs/data-display-avatar--docs

### Reproduction steps

1.
2.
3.
...

### Expected Behaviour

Avatar text should meet the minimum color contrast ratio of 4.5:1 against its background.

### UI5 Web Components for React Version

2.18.0

### UI5 Web Components Version

2.18.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @tandan0813,

the screenshot you shared is too small to properly review. Please double-check and provide a clearer one.

In the meantime, I’ll forward this to the ui5/webcomponents repository, as the Avatar component is maintained there.

**Comment by ndeshev** [comment]
Hello @tandan0813,

I've measured the contrast ratio of the `ui5-avatar ` icons from the link you provided - https://ui5.github.io/webcomponents-react/v2/?path=/docs/data-display-avatar--docs as well as on the playground sample page here - https://ui5.github.io/webcomponents/components/main/Avatar/#icon using the Horizon theme and it matches the accessibility requirements. 

The screenshot is not clearly visible, if there is an additional info you could provide please re-open the issue.

**Comment by tandan0813** [comment]
Hi @ndeshev 
Here is the screenshot, could you please help check it again? ColorScheme is "Placeholder".

<img width="1918" height="1046" alt="Image" src="https://github.com/user-attachments/assets/8d5d34b9-17d7-478a-9ca2-cee09037114f" />

**Comment by ndeshev** [comment]
Hello @UI5/ui5-team-ctr-pir I tested the foreground and background color contrast of the avatars with colors cheme "Placeholder" and it indeed seems to insufficient. The other color schemes are fine, the issue present only with the "Placeholder" one. Could please check it out.

**Comment by kgogov** [comment]
Hello @tandan0813,

I’d like to give you a brief update on your ticket.
We have contacted the responsible colleagues from the design organization, and actions have been initiated to address the accessibility issue. We will share more updates with you soon.

Regards,
Konstantin

**Comment by kgogov** [comment]
Hello @tandan0813,

I’ve rechecked the theme parameters updated from the design theme, and they are now correct.
I’ve attached a screenshot along with the example I tested. I will proceed with closing the issue.

[Example](https://ui5.github.io/webcomponents/play/#eNqdVF1v0zAU_Suen7aqrldgYoR0Ygx4ZRLjASkvrn27eDh2ZDstVdX_vms3gU4MBFEixfdcn3M_cu0d1VbBj1kdG0OLHbWiAVocg1MqnY1gI8LlCWOkNWJ7711nFVs5owhjV5UtTz58vrn7dvuRJFIC0pcYYe8XFQVbUcTSU9YgFK4JvmWIWwPJmBTWxdNCwUpbUGdkh5jSIYUqiHUW3iKwT_ysn_0r5yUoJp1xngn10IX4ZC8p-aCfjAaiILIWPkDElL7efWKXOavelSpHfK1h0zofK0r6uhHcaBXrhYK1lsCyMSXa6qiFYUEKA4v57LwXizpizC-iaQ2U_GBh2byvG5dLp7Ykp4bSSyG_993MlRRkLfwpY0G073-6bpLnrA_wzD9gYPv_kNva6Qsm1iIKTw7dCbKGXN21lFjRHJVK_mvX1b-QXowhvRxDejWGdDGG9HoM6XIM6c0Y0vz8_1i3RkiocR7AP0P8w-z0c1MG6XUbSdy2SapxqjOAhyB4mUyh7ewhZNXDxjzTaZIPM82HY__X6aT76SB1dNkMyPFNwyfHKrVWQCa8srpJZ5NUdMaP3KFrE5wTxMP_Ozdn8IT_DnvDN7CUDhGLMQPH-yby69yuQYnu94_19cPu)

<img width="4366" height="1560" alt="Image" src="https://github.com/user-attachments/assets/a4b5e936-8821-4e56-bf44-9ca439d7a6a0" />
